CHANGELOG

```diff
@@ -0,0 +1,69 @@
+# spark-redshift Changelog
+
+## 4.0.0-SNAPSHOT (2019-06-26)
```

Comment:
The date should be in the version, so that you can release other "snapshot" versions in the future. For example, `4.0.0-snapshot-20190629`

Reply:
Using "pre" might also make your intentions more clear: `4.0.0-pre-release-20190629`. At least, if you told me there were versions `4.0.0` and `4.0.0-snapshot-YYYYMMDD`, I wouldn't necessarily be able to intuit which one came first.

Reply:
Thanks for the Open Source process feedback